lib/uo_cli.rb

```diff
@@ -851,12 +854,20 @@ def self.check_reader
         reopt_post_processor = URBANopt::REopt::REoptPostProcessor.new(scenario_report, scenario_assumptions, scenario_base.reopt_feature_assumptions, DEVELOPER_NREL_KEY)
         if @opthash.subopts[:reopt_scenario] == true
           puts "\nPost-processing entire scenario with REopt\n"
-          scenario_report_scenario = reopt_post_processor.run_scenario_report(scenario_report: scenario_report, save_name: 'scenario_optimization')
+          if @opthash.subopts[:reopt_resilience] == true
```

Comment:
I would just set a variable here if reopt_resilience == true and pass it in the call rather than calling the reopt_post_processor in both the if and else clauses.  easier to maintain.